inflection/tools/dictionary-parser/src/main/java/org/unicode/wikidata/Grammar.java

```diff
@@ -554,11 +537,7 @@ public String toString() {
         }
     }
 
-    public static SortedSet<String> asSet(String... terms) {
-        return new TreeSet<>(Arrays.asList(terms));
-    }
-
-    static final Map<SortedSet<String>, Set<? extends Enum<?>>> TYPEMAP = new HashMap<>(1021);
+    static final Map<String, Set<? extends Enum<?>>> TYPEMAP = new HashMap<>(1021);
```

Comment:
I changed the key type, since Wikidata doesn't include a key set. This is a simplification.